packages/token-vault/src/lib/client.ts

```diff
@@ -9,30 +10,51 @@ interface ClientConfig extends ClientConfigInit {
   proxy: BaseConfig['proxy'];
 }
 
-/**
- * @function client - Initialize the token vault client
- *
- * @param {Object} config
- *
- * @returns {undefined}
+/** ****************************************************************
+ * @function client - Initialize the Token Vault Client
+ * @param {Object} config - The configuration object for the client
+ * @returns {}
+ * @example const register = client({
+ *   app: {
+ *     origin: 'http://app.example.com',
+ *     url: 'http://app.example.com/path',
+ *   },
+ *   forgerock: {
+ *     serverConfig: {
+ *       baseUrl: 'https://openam.forgerock.com/am',
+ *     },
+ *   },
+ *   interceptor: {
+ *     file: 'http://app.example.com/path/interceptor.js',
+ *     scope: '/',
+ *   },
+ *   proxy: {
+ *     origin: 'http://proxy.example.com',
+ *     url: 'http://proxy.example.com/path',
+ *   },
+ * });
  */
-export function client(config: ClientConfig) {
+export function client(config: ClientConfig): ClientInit {
```

Comment:
This is the Token Vault Client initialization code you see in the `main.ts` app within the `e2e/token-vault-app/` directory. This returns the registration object to register all the entities in the system.